README.md

```diff
@@ -9,6 +9,40 @@
 <h1 align="center">Admin Notification Plugin</h1>
 <p align="center">Add custom notification messages in your Sylius admin panel.</p>
 
+## Installation
+
+1. Add the bundle and dependencies in your composer.json :
+
+    ```shell
+    composer require synolia/sylius-admin-notification-plugin --no-scripts
```

Comment:
Can you remove "--no-scripts" because it will use symfony/flex and install it automatically.